docs/project/2024/hadasa.jercau/index.md

```diff
@@ -0,0 +1,112 @@
+# Project Name
+
+PicoW Garden Monitor
```

Comment:
```suggestion
# PicoW Garden Monitor
```

Please add a one line description of the project under the title.

Reply:
hope i managed to solve it, git gave me a hard time